### PR TITLE
ILL SANS reduction fix for python3

### PR DESCRIPTION
**Pull request by gvardany**
**Description of work.**
Fixes string to int conversion for python3.

**To test:**
`SANSILL` system tests pass both on python2 and python3.

*There is no associated issue, no release notes, bug introduced recently.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py [hunk 1]
```diff
@@ -259,7 +259,7 @@ def _process_beam(self, ws):
             att_coeff = run.getLogData('attenuator.attenuation_coefficient').value
         elif run.hasProperty('attenuator.attenuation_value'):
             att_value = run.getLogData('attenuator.attenuation_value').value
-            if att_value < 10 and self._instrument == 'D33':
+            if int(att_value) < 10 and self._instrument == 'D33':
```
Comment (TTitcombe) [inline Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py]: are `att_value` only ever strings of ints? `int("10.0")` doesn't work, for example

Comment (gvardany) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py]: Good question, actually it's a numeric log, I don't know why python3 takes a string as the value.

Comment (gvardany) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py]: The idea is that if it is less than 10, it means it's the index of the attenuator, not the value.

**Review by TTitcombe — APPROVED** [review]
SANSILL system tests pass locally on py2 and py3

**Pull request merged**